pkg/proxy/ipvs/proxier.go

```diff
@@ -716,6 +716,23 @@ func (proxier *Proxier) OnEndpointsSynced() {
 // EntryInvalidErr indicates if an ipset entry is invalid or not
 const EntryInvalidErr = "error adding entry %s to ipset %s"
 
+const ipvsSchedulerKey = "kube-proxy.kubernetes.io/ipvs-scheduler"
+
+var validScheduler = regexp.MustCompile("^(rr|wrr|lc|wlc|lblc|lblcr|dh|sh|sed|nq)$")
```

Comment:
The tricky thing about this is that some IPVS schedulers were introduced in later version of the Linux kernel. Do we need to do a version check for these?

Reply:
IPVS is totally feature gated, so folks would need to opt in, but I agree that there should be a check and fallback.

Reply:
would `sets.String` and `has` be more efficient and easier to read than a regex?

Reply:
Updated.